scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -356,14 +369,26 @@ static void refresh_cpumasks(int idx)
 	trace("LAYER[%d] now has %d cpus, seq=%llu", idx, layer->nr_cpus, layer->cpus_seq);
 }
 
+static void decay_layer_vtime(int layer_idx)
+{
+	struct layer *layer;
+
+	layer = MEMBER_VPTR(layers, [layer_idx]);
+	// TODO: Make the decay value configurable
```

Comment:
IIUC this configurable is going to be how you let less-important tasks sit for say 20 seconds when saturated, right?

Reply:
So there are two parts to this. This decay value is effectively how much history you're considering when deciding whether a task is underserved, this will likely be a whole scheduler argument rather than layer specific, I don't see much benefit to considering different amounts of history for each layer.

The other configurable that we didn't add is in `consume_starved`, where we compare directly to `decay_proportion_time`.  It should be possible to add a fudge factor there per layer that decides how aggressively each layer is scheduled when underserved, but I'm not sure what physical quantity it represents. Probably some sort of variation on "time to wait before being served".